packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/FileUtils.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2011 - 2013 Paul Burke
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This file was modified by the Flutter authors from the following original file:
+ * https://raw.githubusercontent.com/iPaulPro/aFileChooser/master/aFileChooser/src/com/ipaulpro/afilechooser/utils/FileUtils.java
+ */
+
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
You probably only want one license header (the one from the original file)

/cc @Hixie

Reply:
Yeah, don't add a new license here, this file is under the original license.